### ISSUE TITLE
Min-max limit on Automation track

### ISSUE THREAD

**Issue by Lyrcaxis** [LMMS/lmms#4563]
I think it'd be a nice addition to be able to set a min/max to display on the Automation Editor graph.
![Image](https://cdn.discordapp.com/attachments/207159523616489473/484059048136802314/unknown.png)

This'll save users a fair amount of time of scaling and scroll to the desired length and I think it'll be easy to implement.

**Comment by Spekular** [comment]
I agree that this sort of functionality is needed, but it makes more sense
to me that all the zoom levels should be based on fitting the pattern to
the window. Right now Auto does that, but 100%/200%/etc are based on some
seemingly arbitrary scale.

If 100% was what Auto currently is, 200% could be 2X zoom, 400% 4X zoom,
and so on. Specific min/max levels could be nice too, but I think zoom
levels would be easier and faster to use. Thoughts?

**Comment by Lyrcaxis** [comment]
Indeed.. "Vertical Auto" zoom should be doing that for all the graphs! 
(Example; Graph has values of 50-100, "Vertical Auto" should zoom to show 50-100)
As of now "Vertical Auto" appears to be 0.01% zoom for the tempo.
